cloudferrylib/os/identity/keystone.py

```diff
@@ -286,12 +290,18 @@ def _deploy_users(self, users, tenants):
             _user['meta']['new_id'] = self.create_user(user['name'], password,
                                                        user['email'],
                                                        tenant_id).id
-            if self.config['migrate']['keep_user_passwords']:
+
+            if keep_passwd:
                 _user['meta']['overwrite_password'] = True
```

Comment:
This looks counter-intuitive to me. I read this as "If CloudFerry is configured to keep old user passwords, then modify user to have password overwritten (some time later)". Not sure whether it's me or the code wrong.